source/postprocess/heat_flux_map.cc

```diff
@@ -121,6 +132,22 @@ namespace aspect
                         face_area += fe_face_values.JxW(q);
                       }
 
+                    // The opposite face according to the deal.II face indexing
+                    // is always the next or previous face index.
+                    unsigned int opposite_face_index = f;
+                    if (f%2 == 0)
+                      opposite_face_index += 1;
+                    else
+                      opposite_face_index -= 1;
+
+                    fe_face_values.reinit (cell, f);
```

Comment:
> `fe_face_values.reinit (cell, f);`
Should be:
`fe_face_values.reinit (cell, opposite_face_index);`

Reply:
Good catch. Fixed. Does it work for you?